.github/workflows/ci.yml

```diff
@@ -158,6 +158,14 @@ jobs:
     - name: Build and Run QEMU tests
       working-directory: firmware/qemu
       run: ./test.sh
+    - name: Backward compatibility check against decoder v0.2.0
+      working-directory: firmware/qemu
+      env:
+        CARGO_TARGET_THUMBV7M_NONE_EABI_RUNNER: qemu-run
+        QEMU_RUN_IGNORE_VERSION: 1
+      run: |
+        cargo install --debug --git https://github.com/knurling-rs/defmt --rev 7814e1c qemu-run
```

Comment:
nit: could you tag this revision so it's more clear that it is v0.2.0 plus the the new `QEMU_RUN_IGNORE_VERSION`. (I guess next time we'll be able to directly use v0.3.0 here)